README.md

```diff
@@ -34,6 +34,7 @@
 - [Hooks.guide](https://www.hooks.guide/) Collection of React hooks curated by the community.
 - [Searchable Collection of React Hooks](https://nikgraf.github.io/react-hooks/)
 - [useHooks(🐠)](https://usehooks.com/) One new React Hook recipe every day.
+- [hooks-by-example](https://github.com/latviancoder/hooks-by-example) Collection of beginner-friendly real world examples of hook usage.
```

Comment:
Please add it in the first place in order to keep the links sorted alphabetically :)

Reply:
Done